pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -12,11 +12,11 @@
 , proprietaryCodecs ? true
 , enablePepperFlash ? false
 , enableWideVine ? false
-, useVaapi ? false # test video on radeon, before enabling this
+, useVaapi ? true
 , useOzone ? false
 , cupsSupport ? true
 , pulseSupport ? config.pulseaudio or stdenv.isLinux
-, commandLineArgs ? ""
+, commandLineArgs ? "--disable-accelerated-video-decode --disable-accelerated-video-encode"
```

Comment:
```suggestion
# disabled because it might break on radeon, enabled in `chromium-vaapi` variant.
, commandLineArgs ? "--disable-accelerated-video-decode --disable-accelerated-video-encode"
```

Reply:
Thank you!